### PR TITLE
Remove experiment name

### PR DESCRIPTION
**Pull request by ashleve**

### CODE REVIEW

**Comment by RafayAK** [comment]
@ashleve any reason for this change? Does it make checking which logs belong to which experiment kinda hard?

**Comment by ashleve** [comment]
Hi @RafayAK 

Main reason was to just simplify things since there is already task name and tags. 

I agree having the experiment name in logging path makes it easier to find the right logs by browsing folders. But do you find yourself often needing to browse those files manually?

When doing a lot of experiments I found it a bit bothersome to come up with specific names for each, it was easier to mark them with a bunch of tags.

When I want to find my previous experiments, I either filter them by tags using logger dashboard or write code/notebook which retrieves all experiments with given tags.

To sum up I found myself rarely using the experiment name. Do you have a different experience?

**Comment by RafayAK** [comment]
@ashleve I guess I see your point.
